src/main/java/dev/openfeature/javasdk/EvaluationContext.java

```diff
@@ -39,6 +40,10 @@ public <T> T getStructureAttribute(String key, Class<T> klass) {
         return objMapper.readValue(val, klass);
     }
 
+    public Iterator<Map.Entry<String, String>> getStructureAttributes() {
```

Comment:
Mind adding tests as well? That way it stays working. :)

Reply:
Just to be sure, why have you decided to send a `String` and not an `Object` here?

Reply:
Yup I will add some tests right now. 
I sent a string here because thats the way we store it serialized currently. If we wanted to fetch it as an object, we would need to know the type of each of the values. This could be done if we keep track of the types of each value in something like a map `{key -> class of value}`, then we would be able to deserialize each object when fetching all. However I'll leave that up to @justinabrahms